.github/workflows/ci.yaml

```diff
@@ -256,13 +256,17 @@ jobs:
            github.event_name == 'pull_request'
            && !contains(github.event.pull_request.body, '[skip ci]')
            && !contains(github.event.pull_request.body, '[skip community_build]')
+           && !contains(github.event.pull_request.body, '[skip community_build_a]')
          )
          || (
            github.event_name == 'workflow_dispatch'
            && github.repository == 'lampepfl/dotty'
          )"
 
     steps:
+      - name: Set JDK 15 as default
```

Comment:
I don't think the image has JDK 15 either, just JDK 16 and JDK 8 as the default, so this probably doesn't do anything.